src/inspect_ai/_view/www/src/client/api/api-browser.ts

```diff
@@ -190,6 +205,8 @@ async function apiRequest<T>(
   path: string,
   request: Request<T>,
 ): Promise<{ raw: string; parsed: T }> {
+  const url = buildApiUrl(path);
+
   // build headers
   const responseHeaders: HeadersInit = {
```

Comment:
by the way shouldn't these be called `requestHeaders`?

Reply:
yes!